sonic_y_cable/y_cable.py

```diff
@@ -44,6 +57,24 @@
     helper_logger.log_warning("Failed to load chassis due to {}".format(repr(e)))
 
 
+def y_cable_validate_read_data(result, size, physical_port, message):
+
+    if result is not None:
+        if isinstance(result, bytearray):
+            if len(result) != size:
+                helper_logger.log_error("Error: for checking mux_cable {}, eeprom read returned a size {} not equal to 1 for port {}".format(message,
+                                                                                                                                             len(result), physical_port))
```

Comment:
Line breaks are awkward and don't really decrease line size. Suggest the following:

```python
if len(result) != size:
    LOG_MESSAGE_TEMPLATE = "Error: for checking mux_cable {}, eeprom read returned a size {} not equal to 1 for port {}"
    helper_logger.log_error(LOG_MESSAGE_TEMPLATE.format(message, len(result), physical_port))
```

Reply:
fixed